docs/source/blogs/tech_blog/blog13_Inference_Time_Compute_Implementation_in_TensorRT-LLM.md

```
@@ -16,7 +16,7 @@ By NVIDIA TensorRT LLM Team and UCSD Hao AI Lab
     - [Introduction for Dynasor](#dynasor-introduction)
     - [Implement Dynasor-CoT in Scaffolding](#dynasor-cot-implement-in-scaffolding)
     - [Implement Dynasor-CoT based Majority Voting in Scaffolding](#dynasor-cot-based-majority-vote-in-scaffolding)
-    - [Reference](#dynasor-reference)
+    - [Acknowledgements](#dynasor-acknowledgements)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)